src/system_tray.cpp

```diff
@@ -268,5 +276,74 @@ namespace system_tray {
     return 0;
   }
 
+  void
+  update_tray_playing(std::string app_name) {
+    tray.notification_title = NULL;
+    tray.notification_text = NULL;
+    tray.notification_cb = NULL;
+    tray.notification_icon = NULL;
+    tray.icon = TRAY_ICON_PLAYING;
+    tray_update(&tray);
+    tray.icon = TRAY_ICON_PLAYING;
+    tray.notification_title = "Stream Started";
+    char msg[256];
+    sprintf(msg, "Streaming Started for App %s", app_name.c_str());
```

Comment:
The capitalization is somewhat inconsistent for these messages. For the title, let's capitalize each word and for the message, let's capitalize just the first word.

Maybe we could also drop "App" from the message. It might read funny for some app names.